static/src/questionnaires/QuestionnaireBodyCreate.vue

```diff
@@ -64,21 +64,42 @@
                     La question associée à ce thème sera également supprimée.
                   </span>
                   <span v-else>
-                    Les {{ themes[themeIndex].questions.length }} questions associées à ce thème seront également supprimées.
+                    Les {{ themes[themeIndex].questions.length }} questions associées à ce thème
+                    seront également supprimées.
                   </span>
                 </p>
               </confirm-modal>
             </div>
 
             <div v-for="(question, qIndex) in themes[themeIndex].questions"
+                 :id="'theme-' + themeIndex + '-question-' + qIndex"
                  class="card border-0 m-0 pt-2"
                  :key="'question-' + qIndex"> <!-- Card for each question -->
               <div class="card-header border-0">
-                <label v-bind:for="'question' + (themeIndex + 1) + '.' + (qIndex + 1)">
-                  <span class="stamp stamp-md bg-blue mr-3" style="cursor: pointer">
-                    {{ themeIndex + 1 }}.{{ qIndex + 1 }}
-                  </span>
-                </label>
+                <div class="flex-column align-items-center mr-4">
+                  <a :class="{ disabled: qIndex === 0 }"
+                     class="btn btn-secondary btn-sm"
+                     role="button"
+                     href="javascript:;"
```

Comment:
I'm not sure what is the purpose of this <a> tag. But if it is not a link it should just be a simple button to avoid the use  of href="javascript:;".

I need more information to understand this part.

See this post https://ourcodeworld.com/articles/read/209/should-i-use-hash-or-javascript-void-0-on-the-href-attribute-of-my-empty-links

Reply:
done

Reply:
We agreed on using `<button>` with `@click.prevent`.